### PR TITLE
Metadata documentation

### PR DESCRIPTION
**Pull request by meganelacheny**
- Add new index "Configuring metadata"
- Improve Standard metadata documentation
- Create Interoperability + Admin metadata documentations

### CODE REVIEW

File: source/publishing_data/06_configuring_metadata/configuring_metadata.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+.. localizedimage:: images/metadata_configuration_interface.png
+    :alt: Information tab when publishing a new dataset, to configure the metadata of the dataset
+
+
+There are 4 kinds of metadata in the OpenDataSoft platform:
+
+- **Standard metadata** *(default)*: basic metadata displayed in the front office for the users.
+- **Interoperability metadata** *(on demand)*: intended for automatic usage by other systems for interoperability purposes, or for regulatory compliance.
+- **Admin metadata** *(on demand)*: intended for administrators usage only, therefore never displayed anywhere in the front office.
+- **Extra metadata** *(on demand)*: intended for specific applications, and not expected to be used by users directly (e.g. when configuring a default visualizations on the OpenDataSoft portal, this information is stored as an extra metadata). In most cases, the users will never have to access or edit them by themselves. The extra metadata will not be documented in this documentation, please contact OpenDataSoft for more information about these metadata.
```
Comment (remilejeune) [inline source/publishing_data/06_configuring_metadata/configuring_metadata.rst]: Visualizations are stored in **extra metadata**  but it's technical and should change. I think that we do not want to talk about it.

We could talk about some publishing meta and computed metadata like:
- reuse_count
- download_count
- popularity_score
...

File: source/publishing_data/06_configuring_metadata/standard_metadata.rst [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  * * Title
+    * Title of the dataset. Only metadata located above the edition tabs of the interface instead of in the Information tab.
+    * Write a dataset title in the textbox. All special characters are supported. Modifying the title does not change the technical identifier of the dataset.
+  * * Identifier
+    * Technical identifier of the dataset.
+    * 1. Click the "Modify identifier" button.
+      2. Modify the current identifier of the dataset.
+      3. Click the "Modify" button for the changes to be taken into account.
+
+      .. admonition:: Warning
```
Comment (remilejeune) [inline source/publishing_data/06_configuring_metadata/standard_metadata.rst]: We could add another warning: the dataset_id cannot be changed if the dataset is published (we have the question on the support).

File: source/publishing_data/06_configuring_metadata/standard_metadata.rst [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+  * * Last modification date
+    * Date indicating when the dataset was last modified.
+    * The date can either be set manually or automatically.
+
+      * To set the date manually, click the "Set to now" button. Click on this button each time the last modification date needs to be updated.
+      * To set the date automatically, depending on which action should lead to the modification of the date, tick:
+
+        * the "Update automatically when metadata has changed" box,
+        * or, the "Update automatically when data has changed" box.
```
Comment (remilejeune) [inline source/publishing_data/06_configuring_metadata/standard_metadata.rst]: not a real Or, both options are not exclusive.

File: source/publishing_data/06_configuring_metadata/interoperability_metadata.rst [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    * Usage
+  * * Domain
+    * Indicate the domain in the textbox.
+  * * Rights
+    * Indicate the rights in the textbox.
+
+INSPIRE
+-------
+
+INSPIRE metadata are defined by the European Commission. They should ensure that metadata are created for specific spatial data sets and services and that those metadata are kept up to date. These metadata particularly focus on datasets with geographic coordinates.
```
[the remark this answers is not in the document]
Comment (meganelacheny) [reply source/publishing_data/06_configuring_metadata/interoperability_metadata.rst]: As discussed together, here we are talking about "[data] [sets and services]". Also, I took this sentence from the INSPIRE official documentation, so I prefer to leave it as is.

**Pull request merged**